robotbase/src/main/java/frc/robot/Constants.java

```diff
@@ -28,9 +27,9 @@
 public final class Constants {
 
   public static class CAN_ID {
-
     public static final int ELEVATOR_LEFT_MOTOR = 23;
     public static final int ELEVATOR_RIGHT_MOTOR = 24;
+    public static final int ALGAE_RUNNER_MOTOR = 25;
```

Comment:
Let's set all of these can ids to `-1` for now since we don't know what they'll be just yet.